apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClientImpl.ModelCrud.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright 2020 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Api.Gax.Rest;
+using Google.Apis.Bigquery.v2.Data;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using static Google.Apis.Bigquery.v2.ModelsResource;
+
+namespace Google.Cloud.BigQuery.V2
+{
+    public partial class BigQueryClientImpl
+    {
+        // Visible for testing
+        internal sealed class ModelPageManager : IPageManager<ListRequest, ListModelsResponse, BigQueryModel>
+        {
+            private readonly BigQueryClient _client;
+
+            internal ModelPageManager(BigQueryClient client)
```

Comment:
Maybe use an expression-bodied constructor now? (Possibly commit this as is, then we can create another PR that does this for all the page managers...)